test/test_python_type_stubs.py

```diff
@@ -0,0 +1,161 @@
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+import textwrap
+from typing import List
+
+import unittest
+from mock import Mock
+
+from stone.api import Api, ApiNamespace
+from stone.data_type import (
+    Struct,
+    StructField,
+    Boolean,
+    List as StoneList,
+    UInt64,
+    Union,
+    UnionField,
+    Void,
+)
+from stone.target.python_type_stubs import PythonTypeStubsGenerator
+
+class TestPythonTypeStubsHelpers(object):
```

Comment:
This class requires no state, so better to make this a collection of top-level functions.